app/scripts/lib/rpc-method-middleware/handlers/request-accounts.js

```diff
@@ -138,7 +133,7 @@ async function requestEthereumAccountsHandler(
         properties: {
           is_first_visit: isFirstVisit,
           number_of_accounts: Object.keys(metamaskState.accounts).length,
-          number_of_accounts_connected: numberOfConnectedAccounts,
+          number_of_accounts_connected: accounts,
```

Comment:
? 
a) is `accounts` a number, as the property name seems to imply it should be?
b) how does this relate tot he other changes here?

Reply:
a) whoops... it should be a number, but it isn't currently. good catch

Reply:
b) `getAccounts()`, a hook we need for getting accounts ordered correctly, already gets the accounts from the permission. We don't need a separate call to `getPermissionsForOrigin` to also get the accounts (ordered potentially incorrectly, not that it matters). So i got rid of the `getPermissionsForOrigin` call and removed it from the hooks